Terminal.Gui/ConsoleDrivers/CursesDriver/UnixMainLoop.cs

```diff
@@ -119,7 +119,7 @@ void IMainLoopDriver.Setup (MainLoop mainLoop)
 		/// <remarks>
 		///	The token parameter is the value returned from AddWatch
 		/// </remarks>
-		public void RemoveWatch (object token)
+		internal void RemoveWatch (object token)
```

Comment:
You are worry to much on change public to internal on a internal class :-)

Reply:
See my comment above. I want it to be explicit in v2 that external parties to the library can NOT create ConsoleDrivers or IMainLoop implementations.

Reply:
Then why you don't make the `ConsoleDriver` class internal too?